skrub/_agg_joiner.py

```diff
@@ -263,17 +296,24 @@ def fit_transform(self, X, y=None):
         DataFrame
             The augmented input.
         """
-        self._check_inputs(X)
+        if isinstance(self.aux_table, str) and self.aux_table == "X":
```

Comment:
shouldn't those checks be part of check_inputs?

Reply:
I need them before check inputs in case `aux_table` is the placeholder `"X"`, because I assign `_aux_table` then use `CheckInputDataFrame` before `_check_inputs`

But I'm fine moving this whole section that's before `_check_inputs` inside of `_check_inputs`